pytorch_lightning/metrics/classification.py

```diff
@@ -107,12 +107,14 @@ class ConfusionMatrix(TensorMetric):
 
     def __init__(
             self,
+            num_classes: Optional[int] = None,
             normalize: bool = False,
             reduce_group: Any = None,
             reduce_op: Any = None,
```

Comment:
```suggestion
            normalize: bool = False,
            reduce_group: Any = None,
            reduce_op: Any = None,
            num_classes: Optional[int] = None,
```
just for keeping compatibility or is there a strong reason why to have it in front of all arg?

Reply:
it's in front in all other metrics as well.

Reply:
Yeah, I'm just make it like in Accuracy class.

https://github.com/PyTorchLightning/pytorch-lightning/blob/a1ea681c47004599ee5a47a05ddd1b4ea12e60d4/pytorch_lightning/metrics/classification.py#L52-L58

Reply:
yes, ok, hope it does not heart much...
but my point is that if someone is using the func now and pass just argument (not kwargs) it breaks in this new version...